### PR TITLE
Update contact us and add a google map.

### PR DESCRIPTION
**Pull request by mldk-tech**
#60

### CODE REVIEW

**Comment by NofarGizra** [comment]
@DavidGizra 
Please add a screenshot.

**Comment by mldk-tech** [comment]
You see the new google map in the contact us page.

map

**Comment by niryariv** [comment]
something's weird here.. that's not the correct location. What the google maps URL you're using? Did you pan the map before taking the screenshot?

**Comment by mldk-tech** [comment]
I'd use this url:
 https://www.google.com/maps/@-1.27137,36.725557,1605m/data=!3m1!1e3?hl=en

I check what happened

**Comment by mldk-tech** [comment]
contactus

File: ibli/modules/custom/ibli_migrate/html/page/contact-us.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 IBLI Project Development Manager<br />
 International Livestock Research Institute (ILRI)<br />
-P.O. Box 30709 - 00100, Nairobi - Kenya<br />
+P.O. Box 30709 – 00100, Nairobi – Kenya<br />
 Tel: +254-20-422 3000 (SwitchBoard) / 3043 (Direct)<br />
 Fax: +254-20-422 3001<br />
-<br />
-<a href="https://www.google.com/maps/@-1.27137,36.725557,1605m/data=!3m1!1e3?hl=en">MAP</a>
+
+The map at<br />
```
Comment (NofarGizra) [inline ibli/modules/custom/ibli_migrate/html/page/contact-us.html]: Remove the text `The map at<br />`.

**Comment by NofarGizra** [comment]
Merged.

**Pull request closed without merging**